modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/utils/index.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {hierarchy, tree as d3Tree} from 'd3';
+
+import {
+	DX,
+	DY,
+	ID_PROPERTY_NAME_DEFINITIONS,
+	MAX_NAME_LENGTH,
+	ORGANIZATIONS_PROPERTY_NAME,
+	RECT_SIZES,
+	USERS_PROPERTY_NAME_IN_ACCOUNT,
+	USERS_PROPERTY_NAME_IN_ORGANIZATION,
+} from './constants';
+
+let chartNodesCounter = 0;
+
+export function formatItem(item, type) {
+	item.type = type;
+	item.children = [];
+	item.chartNodeNumber = ++chartNodesCounter;
+	item.chartNodeId = getChartNodeId(item);
+
+	if (type === 'account') {
+		item.children.push(
+			...item[USERS_PROPERTY_NAME_IN_ACCOUNT].map(formatUserChild)
+		);
+	}
+
+	if (type === 'organization') {
+		item.children.push(
+			...item[ORGANIZATIONS_PROPERTY_NAME].map(formatOrganizationChild),
+			...item.accountInformation.accounts.map(formatAccountChild),
+			...item[USERS_PROPERTY_NAME_IN_ORGANIZATION].map(formatUserChild)
+		);
+	}
+
+	return item;
+}
+
+export const formatAccountChild = (child) => formatChild(child, 'account');
+export const formatOrganizationChild = (child) =>
+	formatChild(child, 'organization');
+export const formatUserChild = (child) => formatChild(child, 'user');
+
+export function formatChild(child, entityType = null) {
+	if (entityType) {
+		child.type = entityType;
+	}
+	child.chartNodeNumber = ++chartNodesCounter;
+	child.chartNodeId = getChartNodeId(child);
+
+	return child;
+}
+
+export function showChildren(d) {
```

Comment:
What does `d` represent? Variable names should be self-explanatory, please see out [guidelines for abbreviations](https://github.com/liferay/liferay-frontend-projects/blob/dc7d947da927de33e556c7570c007325efa560dc/guidelines/general/naming.md#avoid-abbreviations).

Reply:
Using `d` as a parameter name is a kind of d3 standard to represent a chart node. 
Here you find some examples: https://github.com/d3/d3-hierarchy/blob/v3.0.1/README.md#hierarchy

Anyway I can rename it :) I'm not married with it

Reply:
Fair enough. I guess it does make sense to lean towards d3 conventions in d3-related code :+1: